AppServer/google/appengine/api/taskqueue/taskqueue_distributed.py

```diff
@@ -99,23 +99,20 @@ def __init__(self, app_id, host, port, service_name='taskqueue'):
     self.__nginx_host = host
     self.__nginx_port = port
 
-  def _GetTQLocations(self):
-    """ Gets a list of AppScale TaskQueue servers. """
-    if os.path.exists(TASKQUEUE_LOCATION_FILE):
+  def _GetTQLocation(self):
+    """ Gets an active TaskQueue proxy. """
+    if os.path.exists(TASKQUEUE_PROXY_FILE):
       try:
-        with open(TASKQUEUE_LOCATION_FILE) as tq_file:
-          ips = [ip for ip in tq_file.read().split('\n') if ip]
+        with open(TASKQUEUE_PROXY_FILE) as tq_file:
```

Comment:
Before we were returning a location arrays: should we still do the same to be able to hit multiple load_balancers?